### ISSUE TITLE
No score at the end of the script

### ISSUE THREAD

**Issue by Marooned-MB** [TheRemote/PiBenchmarks#14]
I've run the script on Raspberry Pi B rev 2, it took a while to download dozens of packages and compiling some of them. Here is a full log. Notice no score value at the end.
```
pi@raspberrypi:~ $ sudo curl https://raw.githubusercontent.com/TheRemote/PiBenchmarks/master/Storage.sh | sudo bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 32831  100 32831    0     0  47512      0 --:--:-- --:--:-- --:--:-- 47512
/boot: 197.8 MiB (207337984 bytes) trimmed on /dev/mmcblk0p1
/: 25.6 GiB (27490353152 bytes) trimmed on /dev/mmcblk0p2
Board information: Manufacturer: Raspberry Pi Foundation - Model: Raspberry Pi Model B Rev 2 - Architecture: armv6l - OS: Raspbian GNU/Linux 10 (buster)
Fetching required components ...
Get:1 http://archive.raspberrypi.org/debian buster InRelease [32.6 kB]
Get:2 http://download.zerotier.com/debian/buster buster InRelease [36.8 kB]
Get:3 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
Get:4 https://repos.influxdata.com/debian buster InRelease [4,737 B]
Get:5 https://download.docker.com/linux/raspbian buster InRelease [29.7 kB]
Get:6 https://packages.grafana.com/oss/deb stable InRelease [12.1 kB]
Get:7 http://archive.raspberrypi.org/debian buster/main armhf Packages [348 kB]
Get:8 http://raspbian.raspberrypi.org/raspbian buster/main armhf Packages [13.0 MB]
Get:9 http://download.zerotier.com/debian/buster buster/main armhf Packages [2,690 B]
Get:10 https://repos.influxdata.com/debian buster/stable armhf Packages [926 B]
Get:11 https://download.docker.com/linux/raspbian buster/stable armhf Packages [9,194 B]
Get:12 https://packages.grafana.com/oss/deb stable/main armhf Packages [23.1 kB]
Fetched 13.5 MB in 58s (233 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
curl is already the newest version (7.64.0-4+deb10u1).
dmidecode is already the newest version (3.2-1).
pciutils is already the newest version (1:3.5.2-1).
usbutils is already the newest version (1:010-3).
xxd is already the newest version (2:8.1.0875-5).
The following additional packages will be installed:
  libaio1 libboost-atomic1.67.0 libboost-iostreams1.67.0 libboost-regex1.67.0 libboost-system1.67.0 libboost-thread1.67.0 libgfapi0 libgfrpc0 libgfxdr0 libglusterfs0 libhd21 libibverbs1 libnspr4 libnss3 libnuma1 librados2 librbd1
  librdmacm1
Suggested packages:
  gnuplot gfio python-scipy apmd gsmartcontrol smart-notifier
Recommended packages:
  powermgmt-base ibverbs-providers mailx | mailutils
The following NEW packages will be installed:
  bc fio hdparm hwinfo libaio1 libboost-atomic1.67.0 libboost-iostreams1.67.0 libboost-regex1.67.0 libboost-system1.67.0 libboost-thread1.67.0 libgfapi0 libgfrpc0 libgfxdr0 libglusterfs0 libhd21 libibverbs1 libnspr4 libnss3 libnuma1
  librados2 librbd1 librdmacm1 lshw lsscsi nvme-cli sdparm smartmontools
0 upgraded, 27 newly installed, 0 to remove and 44 not upgraded.
Need to get 18.3 MB of archives.
After this operation, 47.8 MB of additional disk space will be used.
Get:1 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf bc armhf 1.07.1-2 [101 kB]
Get:2 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libaio1 armhf 0.3.112-3 [10.8 kB]
Get:3 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libglusterfs0 armhf 5.5-3 [2,724 kB]
Get:4 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libgfxdr0 armhf 5.5-3 [2,488 kB]
Get:5 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libgfrpc0 armhf 5.5-3 [2,506 kB]
Get:6 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libgfapi0 armhf 5.5-3 [2,524 kB]
Get:7 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libibverbs1 armhf 22.1-1 [43.5 kB]
Get:8 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libnuma1 armhf 2.0.12-1 [22.5 kB]
Get:9 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libboost-iostreams1.67.0 armhf 1.67.0-13+deb10u1 [245 kB]
Get:10 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libboost-regex1.67.0 armhf 1.67.0-13+deb10u1 [430 kB]
Get:11 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libboost-system1.67.0 armhf 1.67.0-13+deb10u1 [229 kB]
Get:12 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libboost-atomic1.67.0 armhf 1.67.0-13+deb10u1 [226 kB]
Get:13 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libboost-thread1.67.0 armhf 1.67.0-13+deb10u1 [260 kB]
Get:14 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libnspr4 armhf 2:4.20-1 [89.6 kB]
Get:15 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libnss3 armhf 2:3.42.1-1+deb10u3 [944 kB]
Get:16 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf librados2 armhf 12.2.11+dfsg1-2.1+rpi1 [2,337 kB]
Get:17 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf librbd1 armhf 12.2.11+dfsg1-2.1+rpi1 [842 kB]
Get:18 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf librdmacm1 armhf 22.1-1 [55.1 kB]
Get:19 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf fio armhf 3.12-2 [449 kB]
Get:20 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf hdparm armhf 9.58+ds-1 [108 kB]
Get:21 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf libhd21 armhf 21.63-3 [627 kB]
Get:22 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf hwinfo armhf 21.63-3 [31.0 kB]
Get:23 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf lshw armhf 02.18.85-0.1 [202 kB]
Get:24 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf lsscsi armhf 0.30-0.1 [43.9 kB]
Get:25 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf nvme-cli armhf 1.7-1 [230 kB]
Get:26 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf sdparm armhf 1.10-1 [111 kB]
Get:27 http://ftp.icm.edu.pl/pub/Linux/distributions/raspbian/raspbian buster/main armhf smartmontools armhf 6.6-1 [454 kB]
Fetched 18.3 MB in 13s (1,462 kB/s)
Selecting previously unselected package bc.
(Reading database ... 53236 files and directories currently installed.)
Preparing to unpack .../00-bc_1.07.1-2_armhf.deb ...
Unpacking bc (1.07.1-2) ...
Selecting previously unselected package libaio1:armhf.
Preparing to unpack .../01-libaio1_0.3.112-3_armhf.deb ...
Unpacking libaio1:armhf (0.3.112-3) ...
Selecting previously unselected package libglusterfs0:armhf.
Preparing to unpack .../02-libglusterfs0_5.5-3_armhf.deb ...
Unpacking libglusterfs0:armhf (5.5-3) ...
Selecting previously unselected package libgfxdr0:armhf.
Preparing to unpack .../03-libgfxdr0_5.5-3_armhf.deb ...
Unpacking libgfxdr0:armhf (5.5-3) ...
Selecting previously unselected package libgfrpc0:armhf.
Preparing to unpack .../04-libgfrpc0_5.5-3_armhf.deb ...
Unpacking libgfrpc0:armhf (5.5-3) ...
Selecting previously unselected package libgfapi0:armhf.
Preparing to unpack .../05-libgfapi0_5.5-3_armhf.deb ...
Unpacking libgfapi0:armhf (5.5-3) ...
Selecting previously unselected package libibverbs1:armhf.
Preparing to unpack .../06-libibverbs1_22.1-1_armhf.deb ...
Unpacking libibverbs1:armhf (22.1-1) ...
Selecting previously unselected package libnuma1:armhf.
Preparing to unpack .../07-libnuma1_2.0.12-1_armhf.deb ...
Unpacking libnuma1:armhf (2.0.12-1) ...
Selecting previously unselected package libboost-iostreams1.67.0:armhf.
Preparing to unpack .../08-libboost-iostreams1.67.0_1.67.0-13+deb10u1_armhf.deb ...
Unpacking libboost-iostreams1.67.0:armhf (1.67.0-13+deb10u1) ...
Selecting previously unselected package libboost-regex1.67.0:armhf.
Preparing to unpack .../09-libboost-regex1.67.0_1.67.0-13+deb10u1_armhf.deb ...
Unpacking libboost-regex1.67.0:armhf (1.67.0-13+deb10u1) ...
Selecting previously unselected package libboost-system1.67.0:armhf.
Preparing to unpack .../10-libboost-system1.67.0_1.67.0-13+deb10u1_armhf.deb ...
Unpacking libboost-system1.67.0:armhf (1.67.0-13+deb10u1) ...
Selecting previously unselected package libboost-atomic1.67.0:armhf.
Preparing to unpack .../11-libboost-atomic1.67.0_1.67.0-13+deb10u1_armhf.deb ...
Unpacking libboost-atomic1.67.0:armhf (1.67.0-13+deb10u1) ...
Selecting previously unselected package libboost-thread1.67.0:armhf.
Preparing to unpack .../12-libboost-thread1.67.0_1.67.0-13+deb10u1_armhf.deb ...
Unpacking libboost-thread1.67.0:armhf (1.67.0-13+deb10u1) ...
Selecting previously unselected package libnspr4:armhf.
Preparing to unpack .../13-libnspr4_2%3a4.20-1_armhf.deb ...
Unpacking libnspr4:armhf (2:4.20-1) ...
Selecting previously unselected package libnss3:armhf.
Preparing to unpack .../14-libnss3_2%3a3.42.1-1+deb10u3_armhf.deb ...
Unpacking libnss3:armhf (2:3.42.1-1+deb10u3) ...
Selecting previously unselected package librados2:armhf.
Preparing to unpack .../15-librados2_12.2.11+dfsg1-2.1+rpi1_armhf.deb ...
Unpacking librados2:armhf (12.2.11+dfsg1-2.1+rpi1) ...
Selecting previously unselected package librbd1:armhf.
Preparing to unpack .../16-librbd1_12.2.11+dfsg1-2.1+rpi1_armhf.deb ...
Unpacking librbd1:armhf (12.2.11+dfsg1-2.1+rpi1) ...
Selecting previously unselected package librdmacm1:armhf.
Preparing to unpack .../17-librdmacm1_22.1-1_armhf.deb ...
Unpacking librdmacm1:armhf (22.1-1) ...
Selecting previously unselected package fio.
Preparing to unpack .../18-fio_3.12-2_armhf.deb ...
Unpacking fio (3.12-2) ...
Selecting previously unselected package hdparm.
Preparing to unpack .../19-hdparm_9.58+ds-1_armhf.deb ...
Unpacking hdparm (9.58+ds-1) ...
Selecting previously unselected package libhd21:armhf.
Preparing to unpack .../20-libhd21_21.63-3_armhf.deb ...
Unpacking libhd21:armhf (21.63-3) ...
Selecting previously unselected package hwinfo.
Preparing to unpack .../21-hwinfo_21.63-3_armhf.deb ...
Unpacking hwinfo (21.63-3) ...
Selecting previously unselected package lshw.
Preparing to unpack .../22-lshw_02.18.85-0.1_armhf.deb ...
Unpacking lshw (02.18.85-0.1) ...
Selecting previously unselected package lsscsi.
Preparing to unpack .../23-lsscsi_0.30-0.1_armhf.deb ...
Unpacking lsscsi (0.30-0.1) ...
Selecting previously unselected package nvme-cli.
Preparing to unpack .../24-nvme-cli_1.7-1_armhf.deb ...
Unpacking nvme-cli (1.7-1) ...
Selecting previously unselected package sdparm.
Preparing to unpack .../25-sdparm_1.10-1_armhf.deb ...
Unpacking sdparm (1.10-1) ...
Selecting previously unselected package smartmontools.
Preparing to unpack .../26-smartmontools_6.6-1_armhf.deb ...
Unpacking smartmontools (6.6-1) ...
Setting up smartmontools (6.6-1) ...
Created symlink /etc/systemd/system/multi-user.target.wants/smartd.service → /lib/systemd/system/smartd.service.
Setting up libibverbs1:armhf (22.1-1) ...
Setting up nvme-cli (1.7-1) ...
Setting up libboost-regex1.67.0:armhf (1.67.0-13+deb10u1) ...
Setting up bc (1.07.1-2) ...
Setting up hdparm (9.58+ds-1) ...
Setting up lshw (02.18.85-0.1) ...
Setting up libglusterfs0:armhf (5.5-3) ...
Setting up sdparm (1.10-1) ...
Setting up libhd21:armhf (21.63-3) ...
Setting up libnspr4:armhf (2:4.20-1) ...
Setting up lsscsi (0.30-0.1) ...
Setting up libboost-iostreams1.67.0:armhf (1.67.0-13+deb10u1) ...
Setting up libnuma1:armhf (2.0.12-1) ...
Setting up libboost-atomic1.67.0:armhf (1.67.0-13+deb10u1) ...
Setting up libboost-system1.67.0:armhf (1.67.0-13+deb10u1) ...
Setting up libaio1:armhf (0.3.112-3) ...
Setting up libgfxdr0:armhf (5.5-3) ...
Setting up librdmacm1:armhf (22.1-1) ...
Setting up libboost-thread1.67.0:armhf (1.67.0-13+deb10u1) ...
Setting up libnss3:armhf (2:3.42.1-1+deb10u3) ...
Setting up hwinfo (21.63-3) ...
Setting up libgfrpc0:armhf (5.5-3) ...
Setting up librados2:armhf (12.2.11+dfsg1-2.1+rpi1) ...
Setting up librbd1:armhf (12.2.11+dfsg1-2.1+rpi1) ...
Setting up libgfapi0:armhf (5.5-3) ...
Setting up fio (3.12-2) ...
Processing triggers for libc-bin (2.28-10+rpi1) ...
Processing triggers for systemd (241-7~deb10u4+rpi1) ...
Processing triggers for man-db (2.8.5-2) ...
Processing triggers for install-info (6.5.0.dfsg.1-4+b1) ...
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package iozone3
Reading package lists... Done
Building dependency tree
Reading state information... Done
build-essential is already the newest version (12.6).
0 upgraded, 0 newly installed, 0 to remove and 44 not upgraded.
Clock speeds: CPU: 700 - Core: 250 - RAM:
Building iozone ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 13493  100 13493    0     0  14055      0 --:--:-- --:--:-- --:--:-- 14055
curl: (23) Failed writing body (4096 != 9326)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 4040k  100 4040k    0     0  1901k      0  0:00:02  0:00:02 --:--:-- 1912k

Building iozone for Linux-arm


Building fileop for Linux-arm


Building the pit_server

System rootfs drive (/) has been detected as /dev/mmcblk0p2 (mmcblk0p2)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  775k  100  775k    0     0   763k      0  0:00:01  0:00:01 --:--:--  764k
System:    Kernel: 5.4.72+ armv6l bits: 32 compiler: gcc v: 9.3.0 Console: tty 0
           Distro: Raspbian GNU/Linux 10 (buster)
Machine:   Type: ARM Device System: Raspberry Pi Model B Rev 2 details: BCM2835 rev: 000e
           serial: <filter>
CPU:       Info: Single Core model: ARMv6-compatible v6l variant: arm1176jzf-s bits: 32 type: UP
           arch: v6l rev: 7
           features: Use -f option to see features bogomips: 697
           Speed: 700 MHz max: 700 MHz Core speed (MHz): 1: 700
Graphics:  Device-1: bcm2708-fb driver: bcm2708_fb v: kernel bus ID: N/A
           Device-2: bcm2835-hdmi driver: N/A bus ID: N/A
           Display: server: No display server data found. Headless machine? tty: N/A
           Message: Unable to show advanced data. Required tool glxinfo missing.
Network:   Device-1: Standard Microsystems SMSC9512/9514 Fast Ethernet Adapter type: USB
           driver: smsc95xx bus ID: 1-1.1:3
           IF: eth0 state: up speed: 100 Mbps duplex: full mac: <filter>
           IF-ID-1: br-df722fc9d758 state: down mac: <filter>
           IF-ID-2: docker0 state: down mac: <filter>
           IF-ID-3: ztbpad4u67 state: unknown speed: 10 Mbps duplex: full mac: <filter>
Drives:    Local Storage: total: 29.72 GiB used: 2.88 GiB (9.7%)
           ID-1: /dev/mmcblk0 model: SE32G size: 29.72 GiB
           Message: No Optical or Floppy data was found.
Partition: ID-1: / size: 28.99 GiB used: 2.83 GiB (9.8%) fs: ext4 dev: /dev/mmcblk0p2
           ID-2: /boot size: 252 MiB used: 53.9 MiB (21.4%) fs: vfat dev: /dev/mmcblk0p1
Info:      Processes: 87 Uptime: 25d 1h 35m Memory: 495.8 MiB used: 298.4 MiB (60.2%) gpu: 64 MiB
           Init: systemd runlevel: 5 Compilers: gcc: 8.3.0 Packages: 640 Client: Sudo v: 1.8.27
           inxi: 3.2.01
Card CSD status register: MID: 3 OID: SD PNM: SE32G PRV: 8.0 MDATE: 8/2018
Card SCR status register: SD Physical Version Specification: 5
MicroSD information: Clock Speed: 50.0 - Manufacturer: SanDisk - Model: SE32G - Vendor: SD - Product: SD - HW Version: 0x8 - FW Version: 0x0 - Date Manufactured: 08/2018
Class: A1 Class 10 V30 U3
Running HDParm tests ...
/dev/mmcblk0p2:
 Timing O_DIRECT cached reads:    44 MB in  2.08 seconds =  21.13 MB/sec
 Timing O_DIRECT disk reads:  64 MB in  3.04 seconds =  21.07 MB/sec
HDParm: 21.07 MB/s - HDParmCached: 21.13 MB/s
Running dd tests ...
81920+0 records in
81920+0 records out
335544320 bytes (336 MB, 320 MiB) copied, 18.684 s, 18.0 MB/s
DD Write Speed: 18.0 MB/s
Running fio write test ...
Running fio read test ...
FIO results - 4k RandWrite:  IOPS ( KB/s) - 4k RandRead:  IOPS ( KB/s)
Running iozone test ...
        Iozone: Performance Test of File I/O
                Version $Revision: 3.489 $
                Compiled for 32 bit mode.
                Build: linux-arm

        Contributors:William Norcott, Don Capps, Isom Crawford, Kirby Collins
                     Al Slater, Scott Rhine, Mike Wisner, Ken Goss
                     Steve Landherr, Brad Smith, Mark Kelly, Dr. Alain CYR,
                     Randy Dunlap, Mark Montague, Dan Million, Gavin Brebner,
                     Jean-Marc Zucconi, Jeff Blomberg, Benny Halevy, Dave Boone,
                     Erik Habbinga, Kris Strecker, Walter Wong, Joshua Root,
                     Fabrice Bacchella, Zhenghua Xue, Qin Li, Darren Sawyer,
                     Vangel Bojaxhi, Ben England, Vikentsi Lapa,
                     Alexey Skidanov, Sudhir Kumar.

        Run began: Mon Dec 28 12:12:45 2020

        Auto Mode
        Include fsync in write timing
        O_DIRECT feature enabled
        File size set to 81920 kB
        Record Size 4 kB
        Command line used: iozone/src/current/./iozone -a -e -I -i 0 -i 1 -i 2 -s 80M -r 4k
        Output is in kBytes/sec
        Time Resolution = 0.000001 seconds.
        Processor cache size set to 1024 kBytes.
        Processor cache line size set to 32 bytes.
        File stride size set to 17 * record size.
                                                              random    random     bkwd    record    stride
              kB  reclen    write  rewrite    read    reread    read     write     read   rewrite      read   fwrite frewrite    fread  freread
           81920       4     1803     1985     3862     3918     3768     2143

iozone test complete.
RandRead: 3768 - RandWrite: 2143 - Read: 3862 - Write: 1803
Enter a description of your storage and setup (Example: Kingston A400 SSD on Pi 4 using StarTech SATA to USB adapter)
Description: SanDisk 32GB A1 Class 10 V30 U3 @ Raspberry Pi B rev 2
(Optional) Enter alias to use on benchmark results.  Leave blank for completely anonymous.
Alias (leave blank for Anonymous): MB

(standard_in) 1: syntax error
(standard_in) 2: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error

     Category                  Test                      Result
HDParm                    Disk Read                 21.07 MB/s
HDParm                    Cached Disk Read          21.13 MB/s
DD                        Disk Write                18.0 MB/s
FIO                       4k random read             IOPS ( KB/s)
FIO                       4k random write            IOPS ( KB/s)
IOZone                    4k read                   3862 KB/s
IOZone                    4k write                  1803 KB/s
IOZone                    4k random read            3768 KB/s
IOZone                    4k random write           2143 KB/s

                          Score:

Compare with previous benchmark results at:
https://storage.jamesachambers.com/
pi@raspberrypi:~ $
```

**Comment by TheRemote** [comment]
That is very strange.  Your test actually did everything right up until that point including every test category, inxi, all of it.

It has to be these lines doing it:

```
# Calculate score
Score=$(echo "scale=2; $DDWriteResult * 1024" | bc)
ScratchPad=$(echo "scale=2; $fio4kRandReadIOPS * 4" | bc)
Score=$(echo "scale=2; $Score + $ScratchPad" | bc)
ScratchPad=$(echo "scale=2; $fio4kRandWriteIOPS * 10" | bc)
Score=$(echo "scale=2; $Score + $ScratchPad" | bc)
Score=$(echo "scale=2; $Score + $IO4kRead" | bc)
Score=$(echo "scale=2; $Score + $IO4kWrite" | bc)
ScratchPad=$(echo "scale=2; $IO4kRandRead * 4" | bc)
Score=$(echo "scale=2; $Score + $ScratchPad" | bc)
ScratchPad=$(echo "scale=2; $IO4kRandWrite * 10" | bc)
Score=$(echo "scale=2; $Score + $ScratchPad" | bc)
Score=$(echo "scale=0; $Score / 100" | bc)
```

It says you got the bc program in your apt output.  This is very strange.  The original Pi model should be supported.

Does this happen with the latest version still?

**Comment by Marooned-MB** [comment]
Looks like there is the same issue:
```
iozone test complete.
RandRead: 4080 - RandWrite: 2530 - Read: 4273 - Write: 1726
Enter a description of your storage and setup (Example: Kingston A400 SSD on Pi 4 using StarTech SATA to USB adapter)
Description: SanDisk 32GB A1 Class 10 V30 U3 @ Raspberry Pi B rev 2
(Optional) Enter alias to use on benchmark results.  Leave blank for completely anonymous.
Alias (leave blank for Anonymous): MB

(standard_in) 1: syntax error
(standard_in) 2: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error
(standard_in) 1: syntax error

     Category                  Test                      Result
HDParm                    Disk Read                 21.06 MB/s
HDParm                    Cached Disk Read          21.16 MB/s
DD                        Disk Write                15.9 MB/s
FIO                       4k random read             IOPS ( KB/s)
FIO                       4k random write            IOPS ( KB/s)
IOZone                    4k read                   4273 KB/s
IOZone                    4k write                  1726 KB/s
IOZone                    4k random read            4080 KB/s
IOZone                    4k random write           2530 KB/s

                          Score:

Compare with previous benchmark results at:
https://pibenchmarks.com/
```

**Comment by TheRemote** [comment]
This should be fixed now on the latest revisions -- let me know if you encounter this!

**Comment by Marooned-MB** [comment]
I assume RPi1 is not suitable for such stress.
I run the script again and I had to cut off the power as RPi froze here:
```
HDParm: 21.23 MB/s - HDParmCached: 21.24 MB/s
Running dd tests ...
81920+0 records in
81920+0 records out
335544320 bytes (336 MB, 320 MiB) copied, 20.0735 s, 16.7 MB/s
DD Write Speed: 16.7 MB/s
Running fio write test ...
```

I guess it's no point supporting this old board anymore. Thanks.

**Comment by TheRemote** [comment]
I honestly doubt it's frozen.  We've had 5 successful tests here:

https://pibenchmarks.com/board/Raspberry_Pi_Model_B_Rev_1

The scores are hilariously low.  They are something like 200-400 points.  It probably will take hours to complete.  If you want to leave it overnight I bet it will complete after hours of testing!

**Comment by Marooned-MB** [comment]
Many thanks for the detailed info!

Yeah, I won't be running it overnight as I'm having all my "smart home" stuff there (pihole, influxdb, node-red and few others) and this cuts off DNS for the whole house plus does not grab all the data from sensors, energy meter, etc as the only core is busy with benchmark so yeah, I'll pass. It's enough to know it's slow ;-) (at least I have some high end SD card, the least I could do).

I was trying to get RPi4 but yeah.. maybe around 2023 or so ;-)

Thanks again!
Cheers

**Comment by TheRemote** [comment]
No worries at all, thank you for investigating this and coming back after I closed it so much later.

I'm sorry I didn't catch this earlier.  I was just about to pull out my Pi 1 for real and test it before I looked at the scores and remembered that yeah if it's a device scoring less than 500 points or so that it takes a *very* long time to complete.

If there were very many devices that were in this category I would lower the amount of data the test uses.  The reason I didn't do this is because anything past the Pi 1 will finish the test in only a few minutes or so but below 500 points it gets exponentially higher.  I would have to sacrifice some accuracy on the vast, vast majority of tests for that use case basically to explain my reasoning at a technical level here.

The collaboration is much appreciated.  There's a very nasty Pi shortage right now (it's actually hurting my web traffic a *lot* too because nobody can get new Pis right now).  I'm sure you will love the Pi 4 once it becomes available for a reasonable price (I think it's like $200 on Amazon right now scalped which NOBODY should pay that).  Thanks again!
